hw/top_earlgrey/rdc/chip_earlgrey_asic_scenario.tcl

```diff
@@ -12,9 +12,31 @@
 
 
 # POR_N
-set_reset_scenario { {POR_N {reset { @t0 0 } { #10 1}}}} -name ScnPOR
+# When POR_N released, POK remains high (already released from reset)
+set_reset_scenario { \
+  { POR_N           { reset {#2 0} { #10 1} }} \
+  { u_ast.vcaon_pok { constraint {@t0 1} }} \
+} -name ScnPOR
 
 # AST POK
+set_reset_scenario { \
+  { u_ast.vcaon_pok { reset {@t0 0} { #2 1} } } \
```

Comment:
technically when vcaon_pok asserts, vcmain_pok should also assert. Is that something that needs to be codified? or just defining the root is enough?

Reply:
We can do that too. I will revise that part.